ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2748,6 +2748,40 @@ def test_jax_numpy_polyint(
     )
 
 
+# polydiv
+@handle_frontend_test(
+    fn_tree="jax.numpy.polydiv",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        min_num_dims=1,
+        max_num_dims=1,
```

Comment:
I think `u` and `v` can be multidimensional arrays, so the test should not only be checking for `max_num_dims=1`.

Reply:
I removed `max_num_dims` limit but there is a problem with dtype or tolerance. I checked `polymul` , `polysub` and `polyadd`, followed exact logic of original jax implementation. Can't find why tests are failing.

Reply:
The error I see right now in the CI is due to the data type not being the correct one. You can solve this by casting the result.